src/platform-implementation-js/dom-driver/gmail/gmail-driver/show-custom-thread-list.ts

```diff
@@ -249,7 +249,16 @@ const setupSearchReplacing = (
           threadDescriptors = [];
         }
 
+        const visited = new Set<InitialPair>();
```

Comment:
```suggestion
        const visited = new Set<InitialIDPair>();
```

Should patch up tests.

Reply:
You'll need to run `tsc` locally to patch up the types here.

tsc output as of [3b76776](https://github.com/InboxSDK/InboxSDK/pull/1145/commits/3b76776141fb92d13ac0559f01dff3cfa6d09ad2)

```
src/platform-implementation-js/dom-driver/gmail/gmail-driver/show-custom-thread-list.ts:255:29 - error TS2345: Argument of type 'ThreadDescriptor' is not assignable to parameter of type 'InitialIDPair'.
  Type 'string' is not assignable to type 'InitialIDPair'.

255             if (visited.has(threadDescriptor)) {
                                ~~~~~~~~~~~~~~~~

src/platform-implementation-js/dom-driver/gmail/gmail-driver/show-custom-thread-list.ts:259:25 - error TS2345: Argument of type 'ThreadDescriptor' is not assignable to parameter of type 'InitialIDPair'.

259             visited.add(threadDescriptor);
                            ~~~~~~~~~~~~~~~~


Found 2 errors in the same file, starting at: src/platform-implementation-js/dom-driver/gmail/gmail-driver/show-custom-thread-list.ts:255


Exited with code exit status 2
```